packages/react/fast-components-react-base/src/data-grid/data-grid.stories.tsx

```diff
@@ -0,0 +1,277 @@
+import { storiesOf } from "@storybook/react";
+import React from "react";
+import { isEqual, omit } from "lodash-es";
+import { Omit } from "utility-types";
+import DataGrid, {
+    DataGridCellProps,
+    DataGridColumnDefinition,
+    DataGridProps,
+} from "./index";
+
+function imageCellFn(
+    props: DataGridCellProps,
+    className: string,
+    cellId: React.ReactText,
+    rootElement: React.RefObject<any>,
+    focusTarget: React.RefObject<any>,
+    unhandledProps: object
+): React.ReactNode {
+    return (
+        <div
+            {...unhandledProps}
+            data-cellid={cellId}
+            className={className}
+            ref={rootElement}
```

Comment:
it seems like the developer is being asked to do a lot of work to implement this, can you add a helper wrapper that can be used?

Reply:
I reduced the number of required props and the component will autogenerate basic column definitions if not provided